console/billing/how-to/change-billing-information.mdx

```diff
@@ -0,0 +1,38 @@
+---
+meta:
+  title: How to change your billing information
+  description: This page explains how to change your billing information
+content:
+  h1: How to change your billing information
+  paragraph: This page explains how to change your billing information
+tags: billing payment change address
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+Before you can order Scaleway resources, you must add your billing information to your account.
```

Comment:
```suggestion
Before you can order Scaleway resources, you must add your billing information to your account.
```
I think this should be a different sentence since this is not when you add the billing contact, but when you change it

Reply:
also, maybe since it seems you can only change the billing address, we can maybe change the name of the how-to? do "how to change your billing address"